liquid

```diff
@@ -1,19 +1,6 @@
 #!/bin/bash
 set -e
 
-# cd to file
 cd "$(dirname ${BASH_SOURCE[0]})"
 
-# delete venv if outdated
-if ! shasum -cq venv/requirements.shasum; then
-  rm -rf venv
-fi
-
-# create venv if needed
-if [ ! -d ./venv ]; then
-  python3 -m venv venv
-  ./venv/bin/pip install -r requirements.txt
-  shasum requirements.txt > venv/requirements.shasum
-fi
-
-exec ./venv/bin/python liquid.py "$@"
+pipenv run ./liquid.py $@
```

Comment:
Please write that last bit as `"$@"`, it quotes the arguments, in case they contain space.